packages/core/database/repositories/health-check-repository-factory.js

```diff
@@ -1,27 +1,22 @@
 const { HealthCheckRepositoryMongoDB } = require('./health-check-repository-mongodb');
 const { HealthCheckRepositoryPostgreSQL } = require('./health-check-repository-postgres');
+const { prisma } = require('../prisma');
```

Comment:
Ideally there's no defaulting.
This makes things harder when we want to refactor for a injection container, we might not get the same behaviour when we start injecting things into this class.